.github/workflows/dvc-diff.yml

```diff
@@ -0,0 +1,53 @@
+# Checks for image diffs in a Pull Request and adds a GitHub comment showing the diff
+name: DVC image diff
+
+on:
+  pull_request:
+    paths:
+      - 'pygmt/tests/baseline/*.png.dvc'
+
+jobs:
+  dvc-diff:
+    name: DVC image diff
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        # fetch all history so that git diff works
+        fetch-depth: 0
+
+    - name: Setup data version control (DVC)
+      uses: iterative/setup-dvc@v1
+
+    - name: Setup continuous machine learning (CML)
+      uses: iterative/setup-cml@v1
+
+    # Produce the markdown diff report, which should look like:
+    # | Status   | Path                                |
+    # |----------|-------------------------------------|
+    # | added    | pygmt/tests/baseline/test_image.png |
+    - name: Get list of images that were added or changed
+      run: |
+        dvc diff --show-md master HEAD >> report.md
```

Comment:
Typo?
```suggestion
        dvc diff --show-md master HEAD > report.md
```

Reply:
I've added another header line above (`## Summary of changed images`) so this change isn't needed anymore.